kenichi127/PoE
# Answer　Page

## Description / Purpose (checkpoints)
We can see answer and share by SNS

Fixes issue: https://docs.google.com/spreadsheets/d/1BHy8K_EvpG4fBLjujxShiZWe2-tNs0Jj/edit#gid=905621989
task:13

## How To Test This PR
 1. input many data in daydata(need past date and one future date(2020/1/1).We need same value of them. )
 2. select date on homepage (2020/1/1)
 3. Click search
4.We can see answer page.
5.Click each button(LINE,Twitter,Facebook)
6.We can share URL and answer data.
 
 **Commands**
php artisan migrate

 **Screenshots**
(answer page)
![image](https://user-images.githubusercontent.com/49220772/59327618-a6e2e880-8d1c-11e9-80a7-525f29c1b8a9.png)
(LINE share)
![image](https://user-images.githubusercontent.com/49220772/59327646-b7935e80-8d1c-11e9-9e17-114d4d9bfa36.png)
(Facebook share)
![image](https://user-images.githubusercontent.com/49220772/59327662-c7ab3e00-8d1c-11e9-8df6-d64d9a9210da.png)
(Twitter share)
![image](https://user-images.githubusercontent.com/49220772/59327675-d560c380-8d1c-11e9-96d9-59f713b923fb.png)
